modules/bastion/compute.tf

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2019, 2023 Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
+
+locals {
```

Comment:
Can you please keep the same file organization that we are using in https://github.com/oracle-terraform-modules/terraform-oci-bastion?